credential/exchange/verification.go

```diff
@@ -115,13 +116,19 @@ func VerifyPresentationSubmissionVP(def PresentationDefinition, vp credential.Ve
 		}
 
 		// resolve the claim from the JSON path expression in the submission descriptor
-		submittedClaim, err := jsonpath.JsonPathLookup(vpJSON, submissionDescriptor.Path)
+		claim, err := jsonpath.JsonPathLookup(vpJSON, submissionDescriptor.Path)
 		if err != nil {
 			err := errors.Wrapf(err, "could not resolve claim from submission descriptor<%s> with path: %s",
 				submissionDescriptor.ID, submissionDescriptor.Path)
 			logrus.WithError(err).Error()
 			return err
 		}
+		submittedClaim, err := getClaimAsJSON(claim)
+		if err != nil {
+			err := errors.Wrapf(err, "getting claim as go-json: <%s>", claim)
+			logrus.WithError(err).Error()
+			return err
```

Comment:
`util.NewErrorf()`...

Reply:
Keeping it here for consistency, and preventing scope creep. This should be addressed in https://github.com/TBD54566975/ssi-sdk/issues/258